app/controllers/events_controller.rb

```diff
@@ -35,6 +35,8 @@ def invalid_request(message = 'invalid request', type: 'invalid_request')
   end
 
   def verify_signature
+    return true if Rails.env.development?
```

Comment:
Wouldn't this bypass the ability to test whether this method will actually do the needful once we ship it?